### PR TITLE
fix: Add support for aliases and grants on `aws_kms_external_key`

### PR DESCRIPTION
**Pull request by ORuessel**
Add to support aliases and permissions for aws_kms_external

## Description

In first test it is not possible to add aliases and the permissions to aws_kms_external_key. 

## How Has This Been Tested?
- [ ] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [ ] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -281,3 +281,5 @@ resource "aws_kms_grant" "this" {
   grant_creation_tokens = try(each.value.grant_creation_tokens, null)
   retire_on_delete      = try(each.value.retire_on_delete, null)
 }
+
```
Comment (bryantbiggs) [inline main.tf:284]: please remove these extra lines for the CI checks to pass

Comment (ORuessel) [reply main.tf]: @bryantbiggs thanks for the information. I has remove the extra lines. Best regards

**Review by bryantbiggs — APPROVED** [review]
thanks @ORuessel !

**Comment by antonbabenko** [comment]
This PR is included in version 1.0.3 :tada:

**Pull request merged**